plugins/field-slider/test/field_slider_test_helpers.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
Is the idea that these helpers would end up in dev-tools eventually? If so, I think the name of all these methods needs to be more generic.

Reply:
Only the first one I was initially intending to be in dev-tools, the others are currently too specific to slider/fieldnumber.

Reply:
To keep track of things easier, I've made a field_slider_helpers and a field_helpers now